sdk/src/main/scala/com/horizen/account/utils/AccountBlockFeeInfo.scala

```diff
@@ -27,10 +27,18 @@ object AccountBlockFeeInfoSerializer extends SparkzSerializer[AccountBlockFeeInf
   }
 
   override def parse(r: Reader): AccountBlockFeeInfo = {
+    var bigIntBitLength: Integer = 0
```

Comment:
here we may remove the check because it is an internal object, so node creates and holds it by itself.
Let's keep the check only in the objects that can be retreived from outside.